app/views/hostgroups/_form.html.erb

```diff
@@ -26,8 +26,16 @@
       <%= text_f f, :name %>
       <%= textarea_f f, :description, :rows => :auto %>
 
-      <%= select_f f, :compute_resource_id, accessible_resource(@hostgroup, :compute_resource), :id, :to_label,
-          { :include_blank => blank_or_inherit_f(f, :compute_resource, blank_value: _('Bare Metal')) },
+      <%= select_f f, :compute_resource_id, 
+          accessible_resource(@hostgroup, :compute_resource), 
+          :id, 
+          :to_label,
+          { 
+            :disable_button_enabled => hostgroup_inherited_by_default?(:compute_resource_id, @hostgroup),
```

Comment:
Do we need to hide this for parent hostgroups?

Reply:
No since they can also be children

Reply:
What about for top-level host groups which don't have a parent?

Reply:
We could hide it, but it isn't hidden in hosts if they have no host group